### DISCUSSION TITLE
Bad Spelling

### DISCUSSION THREAD

**Discussion by paul-bouvier-collectifenergie** [badeball/cypress-cucumber-preprocessor#1124] [General]
There's a spelling mistake in the configuration documentation.

// .cypress-cucumber-preprocessorrc.json
{
  "e2e": {
    "stepDefintions": "cypress/e2e/[filepath].{js,mjs,ts,tsx}"
  },
  "component": {
    "stepDefintions": "src/[filepath].{js,mjs,ts,tsx}"
  }
}

It's stepDefinitions not stepDefintions

**Comment by badeball** [comment]
Good catch! I've fixed it with 6cbf333.
